docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -55,16 +53,44 @@ direct path, and joining with the root parent docs. Note that any
 fields referenced inside the query must use the complete path (fully
 qualified).
 
-The `score_mode` allows to set how inner children matching affects
+The `score_mode` allows you to set how inner children matching affects
 scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
 `max` and `none`.
 
-There is also an `ignore_unmapped` option which, when set to `true` will
+There is also an `ignore_unmapped` option, which when set to `true` will
 ignore an unmapped `path` and will not match any documents for this query.
 This can be useful when querying multiple indexes which might have different
 mappings. When set to `false` (the default value) the query will throw an
 exception if the `path` is not mapped.
 
-Multi level nesting is automatically supported, and detected, resulting
-in an inner nested query to automatically match the relevant nesting
+==== Multi-level nesting
+
+Multi-level nesting is automatically supported and detected, resulting
+in an inner nested query that automatically matches the relevant nesting
 level (and not root) if it exists within another nested query.
+
+To set up a multi-level nested query, use this example. Notice that each level within the nest is separated by a dot (name.state.city).
+
+[source,js]
+--------------------------------------------------
+GET /_search
+{
+    "query": {
+        "nested": {
+            "path": "name",
+            "query": {
+                "nested" : {
+                    "path" :  "name.state",
+                    "query" :  {
+                       "match": {
+                           "name.state.city": "Atlanta"
+                        }
+                    }
+                }
+            },
+            "inner_hits": {}
+        }
+    }
+}
+--------------------------------------------------
+// CONSOLE
```

Comment:
You'll need to set up a mapping with a compatible field for this to work with the tests. You may as well write the entire example in the code snippet, including configuring the mapping and adding sample data. These things are fairly complex to set up and showing folks *all* of the set up is pretty nice for this.

Reply:
Ok, I've made a lot of changes, making the first example one from the nested datatype (which I thought was simpler to understand) and expanding the multi-level query example. I think this could use simplifying as well, but I'm too new to figure out how! Adding martijnvg for review.

Reply:
Added a more complete example for multi-level query and changed original example to match the nested datatype example, as it is simpler and might not be seen by the user. Made additional changes to text and added martijnvg as reviewer.